packages/sdk/client/src/packlets/diagnostics/monitor.test.ts

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { expect } from 'chai';
+
+import { describe, test } from '@dxos/test';
+import { range } from '@dxos/util';
+
+export type TimeRecord = { timestamp: number };
+export type TimeBucket = { start: number; period: number; count: number };
+
+export const createBucketReducer =
```

Comment:
E.g., here's a trivial example of a reducer we might want to call on-the-fly from console and/or devtools.